src/share/classes/com/sun/javatest/regtest/agent/Alarm.java

```diff
@@ -143,9 +143,9 @@ public boolean didFire() {
     protected void run() {
         if (msgOut != null) {
             if (count == 0) {
-                msgOut.println(String.format("Timeout refired %s times", TimeUnit.SECONDS.convert(delay, delayUnit)));
+                msgOut.println(String.format("Timeout signalled after %d seconds", TimeUnit.SECONDS.convert(delay, delayUnit)));
             } else if (count % 100 == 0) {
-                msgOut.println(String.format("Timeout signalled after %d seconds", count));
+                msgOut.println(String.format("Timeout refired %s times", count));
```

Comment:
`%s` could be turned into a `%d` here, as `count` is of type `int`.

Reply:
done